diff -up libgit2-glib-1.2.0/libgit2-glib/ggit-types.h.omv~ libgit2-glib-1.2.0/libgit2-glib/ggit-types.h
--- libgit2-glib-1.2.0/libgit2-glib/ggit-types.h.omv~	2024-04-18 18:32:17.546334052 +0200
+++ libgit2-glib-1.2.0/libgit2-glib/ggit-types.h	2024-04-18 18:33:05.758830054 +0200
@@ -341,6 +341,7 @@ typedef enum
  * @GGIT_CONFIG_LEVEL_XDG: XDG compatible configuration file (.config/git/config).
  * @GGIT_CONFIG_LEVEL_GLOBAL: User-specific configuration file, also called Global configuration file.
  * @GGIT_CONFIG_LEVEL_LOCAL: Repository specific configuration file.
+ * @GGIT_CONFIG_LEVEL_WORKTREE: Worktree specific configuration file.
  * @GGIT_CONFIG_LEVEL_APP: Application specific configuration file; freely defined by applications.
  * @GGIT_CONFIG_LEVEL_HIGHEST: Represents the highest level of a config file.
  *
@@ -355,7 +356,8 @@ typedef enum
 	GGIT_CONFIG_LEVEL_XDG         = 3,
 	GGIT_CONFIG_LEVEL_GLOBAL      = 4,
 	GGIT_CONFIG_LEVEL_LOCAL       = 5,
-	GGIT_CONFIG_LEVEL_APP         = 6,
+	GGIT_CONFIG_LEVEL_WORKTREE    = 6,
+	GGIT_CONFIG_LEVEL_APP         = 7,
 	GGIT_CONFIG_LEVEL_HIGHEST     = -1
 } GgitConfigLevel;
 
